application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -56,4 +60,31 @@ public static String escapeMarkdown(String text) {
         return afterEscape.replace("\\```", "\\`\\`\\`");
     }
 
+    /**
+     * Converts a slash command text to a mentioned slash command, which you can directly click on
+     * in Discord.
+     *
+     * @param guild the {@link Guild} that contains the command
+     * @param baseCommandName the command's name
+     * @param subcommands optional subcommands, depending on the base command used
+     * @return Formatted string for the mentioned slash command
+     * @throws IllegalArgumentException when the command name doesn't match with the guild's
+     *         commands
+     */
+    public static RestAction<String> mentionSlashCommand(Guild guild, String baseCommandName,
+            String... subcommands) {
+        List<String> fullCommand = new ArrayList<>(List.of(baseCommandName));
+        fullCommand.addAll(List.of(subcommands));
+        return guild.retrieveCommands().map(guildCommands -> {
+            Command guildCommand = guildCommands.stream()
+                .filter(c -> c.getName().equalsIgnoreCase(baseCommandName))
+                .findAny()
+                .orElseThrow(
+                        () -> new IllegalArgumentException("Command '%s' does not exist in guild %s"
+                            .formatted(String.join(" ", fullCommand), guild.getId())));
```

Comment:
It only checks for the base command

So this should be

```suggestion
                        () -> new IllegalArgumentException("Command '%s' does not exist in guild %s"
                            .formatted(baseCommandName, guild.getId())));
```